courses/machine_learning/deepdive2/image_classification/solutions/1_mnist_linear.ipynb

```diff
@@ -34,7 +34,7 @@
      "name": "stdout",
```

Comment:
<p>Add an extra space than what is there now so that there are 2 spaces after each line before the # for a comment.</p><p> </p><p>Also add spaces in between the equal signs of the first three lines.</p><p> </p><p>Also, [:,DIGIT] -&gt; [:, DIGIT] (space after comma).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>